.changeset/tasty-llamas-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': minor
```

Comment:
there's a semver-purist argument that this should be a minor, but i think it's probably better if it's a patch release?

```suggestion
'@sveltejs/kit': patch
```